### DISCUSSION TITLE
Issue using StreamingResponse

### DISCUSSION THREAD

**Discussion by ryan-wolbeck** [fastapi/fastapi#7127] [Questions] [answered]
Hello,

I tired to make an import similar to below in the docs

https://github.com/tiangolo/fastapi/blob/e77ea635777d2494690ba3eb62bd005b9edeefde/docs_src/custom_response/tutorial007.py#L2

but got the following 
```
Traceback (most recent call last):
File "app/main.py", line 6, in <module>
from fastapi.responses import StreamingResponse
ModuleNotFoundError: No module named 'fastapi.responses'
```

The workaround I used was 
```
from starlette.responses import StreamingResponse
```

I'm using fastapi==0.42.0 within docker

**Comment by ArcLightSlavik** [comment, marked as answer]
` from fastapi.responses import StreamingResponse` works after 0.51.0.
Would suggest upgrading since there's a lot of other stuff you would need to import from `starlette` and not `fastapi` if you stay on 0.42.0.

**Comment by tiangolo** [comment]
Thanks for the help @ArcLightSlavik ! 🍰 

And thanks for closing the issue @ryan-wolbeck ☕ 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
